featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -374,6 +374,8 @@ def update_progress_callback(done):
         feature_matrix.append(_feature_matrix)
 
     else:
+        if schema:
+            cutoff_time.ww.init_with_full_schema(schema=schema)
```

Comment:
@rwedge it doesn't seem like this line is getting hit with the unit test you added.